helm-charts/explorer/templates/explorerdb-configmap.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: explorer-db-env
+  labels:
+    name: {{ $.Values.db.name }}
+    app.kubernetes.io/name: {{ $.Values.db.name }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}  
+  annotations: 
+    {{- toYaml .Values.annotations | nindent 6 }} 
+data:
+  DATABASE_DATABASE: fabricexplorer
+  DATABASE_USERNAME: hppoc
+  DATABASE_PASSWORD: password 
```

Comment:
Again, hardcoding of values,

Reply:
made above values as configurable in values.yaml